README.md

```diff
@@ -1 +1,5 @@
-# cesium-webstorm-plugin
\ No newline at end of file
+# Cesium WebStorm Plugin
```

Comment:
Also provide concise instructions for contributing (really just the CLA) like this: https://github.com/AnalyticalGraphicsInc/gltf-pipeline#contributions

Most likely I don't expect contributions here, but we should welcome them.